test/e2e/suites/managed/cluster.go

```diff
@@ -44,7 +43,7 @@ type ManagedClusterSpecInput struct {
 	E2EConfig                *clusterctl.E2EConfig
 	ConfigClusterFn          DefaultConfigClusterFn
 	BootstrapClusterProxy    framework.ClusterProxy
-	AWSSession               client.ConfigProvider
+	AWSSession               aws.Config
 	AWSSessionV2             *aws.Config
```

Comment:
You'll want to use the `AWSSessionV2` field here in your relevant tests and leave `AWSSession` alone. That seems to be what the current CI errors are complaining about.